libs/workordersidesheet/src/lib/utils-sidesheet/useMccr.ts

```diff
@@ -29,6 +29,7 @@ export const useMccr = (packageId: string | null) => {
   const { data, isFetching, error } = useQuery({
     queryFn: (a) => fetchMccr(ccApp, contextId, packageId, a.signal),
     queryKey: ['mccr', packageId],
+    initialData: [],

```

Comment:
Are you sure this is the appropriate solution?

Reply:
Removed this.